consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/statemachine/IbftRound.java

```diff
@@ -132,11 +138,15 @@ public void handleProposalMessage(final Proposal msg) {
 
     if (updateStateWithProposedBlock(msg)) {
       LOG.debug("Sending prepare message. round={}", roundState.getRoundIdentifier());
-      final Prepare localPrepareMessage =
-          messageFactory.createPrepare(getRoundIdentifier(), block.getHash());
-      transmitter.multicastPrepare(
-          localPrepareMessage.getRoundIdentifier(), localPrepareMessage.getDigest());
-      peerIsPrepared(localPrepareMessage);
+      try {
+        final Prepare localPrepareMessage =
```

Comment:
Is it only the createPrepare that can fail with a SecurityModuleException? Don't think this try/catch needs to be around the peerIsPrepared or multicastPrepare